plugins/workload-management/src/javaRestTest/java/org/opensearch/rest/WorkloadManagementRestIT.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.rest;
+
+import org.apache.hc.core5.http.io.entity.EntityUtils;
+import org.opensearch.client.Request;
+import org.opensearch.client.Response;
+import org.opensearch.client.ResponseException;
+import org.opensearch.test.rest.OpenSearchRestTestCase;
+
+public class WorkloadManagementRestIT extends OpenSearchRestTestCase {
+
+    public void testCreate() throws Exception {
+        Request request = new Request("PUT", "_wlm/query_group");
+        request.setJsonEntity(getCreateJson("analytics", "enforced", 0.4, 0.2));
+        Response response = client().performRequest(request);
+        assertEquals(response.getStatusLine().getStatusCode(), 200);
+        client().performRequest(new Request("DELETE", "_wlm/query_group/analytics"));
```

Comment:
Lets move the logic to create/update/delete/get operations into parametrised methods, this will help us avoid repitions.
e,g;
```
Response performOperation(String method, String uriPath, String json) {
        Request request = new Request(method, uriPath);
        if (json != null) {
              request.setJsonEntity(getCreateJson("analytics", "enforced", 0.4, 0.2)); 
        }
        return client().performRequest(request);
}
```

Now we can use this for CRUD operations like

```
Response getResponse = performOperation("GET", "_wlm/query_group/{name}", null);
Response putResponse = performOperation("GET", "_wlm/query_group/{name}", jsonString);
```